.github/workflows/nm-nightly.yml

```diff
@@ -35,8 +35,8 @@ jobs:
 
             test_configs: '[{"python":"3.8.17","label":"gcp-k8s-l4-solo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"},
                             {"python":"3.9.17","label":"gcp-k8s-l4-solo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"},
-                            {"python":"3.10.12","label":"gcp-k8s-l4-duo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"},
```

Comment:
I assumed that we were splitting `solo` and `duo` across the different versions of python to get a certain amount of coverage.  why change to all `solo`?

Reply:
- none of the tests running right now use the distributed features, so its silly to run them on a multi-gpu instance
- I have two other PRs ongoing to re-enable distributed testing (fan out + enable distributed tests) that will turn this back on